xgfield/fieldsky.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, **kwargs):
         self.gpu     = kwargs.get(    'gpu',fd.gpu)
         self.mpi     = kwargs.get(    'mpi',fd.mpi)
         self.loglev  = kwargs.get( 'loglev',fd.loglev)
-        self.backend = kwargs.get('backend',fd.backend)
```

Comment:
We can still have an external backend. I suggest to leave it in the code.

Reply:
Okay, perhaps we can create an issue to track this and move on...

Reply:
See comment below.